app.py

```diff
@@ -18,21 +17,14 @@ def definedlog(fileHandler):
     return logger
 
 
-def connect_db():
+def connect_db(host, user, password, database):
+    import mysql.connector
```

Comment:
import statement should be in the top of the file. Unless, there is a hidden reason which I am not aware of.